markdown.js

```diff
@@ -0,0 +1,73 @@
+const util = require('./src/tool');
+const fs = require('fs');
+
+const dataset = [
+  { type: 'title', value: 'Headers H1-H6'},
+  { type: 'h1', value: 'H1 Header'},
+  { type: 'h2', value: 'H2 Header'},
+  { type: 'h3', value: 'H3 Header'},
+  { type: 'h4', value: 'H4 Header'},
+  { type: 'h5', value: 'H5 Header'},
+  { type: 'h6', value: 'H6 Header'},
+  { type: 'title', value: 'Lists Ordered, Unordered sub-list, Unordered'},
+  { type: 'list', text: '\n1. One\n2. Two\n3. Three\n4. Four\n5. Five\n'},
```

Comment:
Javascript supports nested arrays. This implementation is currently unacceptable.

Reply:
Like this?
```
{ type: 'list', [ 1, 2, 3, 4, [ one, two, three, four ] ]}
```
but how would I determine which one is ordered and not? and how am I going to figure out if the sub-list is ordered or not?